### DISCUSSION TITLE
Help set up notifications about adding/deleting a table in a Postgres database

### DISCUSSION THREAD

**Discussion by JimmyTodesky** [open-metadata/OpenMetadata#17565] [Q&A]
I can't set up notifications about adding/deleting a table in the database. I did everything according to the instructions in the documentation, but no messages come. Tell me how to track changes in the Postgres database. I set up notifications for changes in the table.

**Comment by Prajwal214** [comment]
@JimmyTodesky Can you share the alert/notification you have configured. Also confirm if the destination is set correctly.

**Reply by JimmyTodesky** [reply]
![image](https://github.com/user-attachments/assets/1e9b6668-23ff-4c93-b1cf-f7da54d74908)
@Prajwal214  This is how I set up notifications. But, when setting up, I tried many options. This is simply the final one.

**Reply by Prajwal214** [reply]
Is it working with different destination? Can you confirm if your webhook accepts needs a specific format for the payload , we only support - slack /msteam/gchat with custom formats currently

**Reply by JimmyTodesky** [reply]
Yes, well at the level of a separate table - when adding columns, everything works. When calling through a browser - the webhook works. I can't set up a notification only when adding a new table to the database.

**Reply by Prajwal214** [reply]
Can you verify in the Open Metadata Server logs if any error related to change event.

**Reply by JimmyTodesky** [reply]
Good day. There are no errors when extracting metadata - I specifically added a table to the DB, launched a metadata update for the DB. After that, the information appeared in openmetadata. However, no webhook notifications were received in Telegram.

`Summary
Basic Configuration
Type
metadata
Schedule
*/5 * * * *
Recent Runs
Success`

**Comment by JimmyTodesky** [comment]
Can you tell me if there is no solution to my situation? Please tell me how to set up notifications about adding/removing tables.

**Reply by Prajwal214** [reply]
In the `Openmetadata` logs can you verify if there are any errors related to change_event table.
